ios/storage.md

```diff
@@ -245,8 +247,6 @@ func getURL() {
           print("Completed: \(data)")
       case .failed(let storageError):
           print("Failed: \(storageError.errorDescription). \(storageError.recoverySuggestion)")
-      case .inProcess(let progress):
-          print("Progress: \(progress)")
       default:
```

Comment:
Curious why we are removing `.inProcess` ?  Maybe it would be better to get rid of the default case instead?

Reply:
the reason why we are removing it is because the Type that is returned for `.inProcess` in this case is Void, because getURL does not emit progress events. And currently all operations are built on top of the AmplifyOperation class which provides events callback with these different cases.

https://github.com/aws-amplify/amplify-ios/blob/461c3cfd8923fbbb6eb10b60495b0f4594879cf4/Amplify/Core/Support/AmplifyOperation.swift#L18

I can see a way to remove the default case is if we create another class like `AmplifyRequestOperation` and another `AsyncEvent` that doesn't have inProcess (https://github.com/aws-amplify/amplify-ios/blob/461c3cfd8923fbbb6eb10b60495b0f4594879cf4/Amplify/Core/Support/AsyncEvent.swift#L18)

this is definitely a good callout regarding having to "default" and know which cases to handle. there have been some discussions around removing the .unknown case

Reply:
adding something over in amplify-ios https://github.com/aws-amplify/amplify-ios/issues/331